Shared/Scripts/Init.cmd

```diff
@@ -111,7 +111,7 @@ REM *********************************
 	REM We'll conditionally set the credential provider if not set on the machine.
 	REM If not set we will set it to the local one in the enlistment but iwth the b-drive substitution
 	if NOT DEFINED NUGET_CREDENTIALPROVIDERS_PATH (
-		set NUGET_CREDENTIALPROVIDERS_PATH=B:\Shared\Tools
+		set NUGET_CREDENTIALPROVIDERS_PATH=TOOLROOT

```

Comment:
>TOOLROOT [](start = 37, length = 8)

Should this be %TOOLROOT%?

Reply:
Why you need to change this? My guess is that we will set NUGET_CREDENTIALPROVIDERS_PATH before calling init, so this code path should not be reached, right?

---
In reply to: [338827815](https://github.com/microsoft/BuildXL/pull/1087#discussion_r338827815) [](ancestors = 338827815)

Reply:
I still think, you don't need this change

---
In reply to: [338828075](https://github.com/microsoft/BuildXL/pull/1087#discussion_r338828075) [](ancestors = 338828075,338827815)

Reply:
This code is reached during the local builds.
Both the paths point to the same place, but TOOLROOT is the actual path where the code is stored and B:\Shared\Tools required the mount to work.
I think TOOLROOT should be safer in general, but I can change it back to the hard coded path B:\Shared\Tools.